src/components/project/project.service.ts

```diff
@@ -514,9 +513,7 @@ export class ProjectService {
     const changes = {
       ...input,
       modifiedAt: DateTime.local(),
-      status: object.step.value
-        ? stepToStatus(object.step.value)
-        : object.status,
+      status: input.step ? stepToStatus(input.step) : object.status,
```

Comment:
Let's skip the change completely by omitting the key:
```suggestion
      ...(input.step ? { status: stepToStatus(input.step) } : {}),
```

Reply:
changed